documentation/otn_redis_schema.md

```diff
@@ -827,86 +842,78 @@ target-attenuation  = float64
 attenuation         = float64  
 ```
 
-### AMPLIFIER
+### AMPLIFIER_TABLE
+
+*;/openconfig-optical-amplifier:optical-amplifiers/amplifier/state*
 
-*openconfig-optical-amplifier*
+*;revision "2019-12-06" {reference "0.5.0"}*
 
 ```
-key                 = AMPLIFIER|<name>      ; string
+key                 = AMPLIFIER_TABLE|<name>      ; string
```

Comment:
follow sonic existing practice:
- Tables in state-db and app-db  are named by adding _TABLE as postfix to the corresponding table in config-db.
see: https://github.com/sonic-net/sonic-swss-common/blob/master/common/schema.h